__tests__/config/jestGlobalSetup.ts

```diff
@@ -216,13 +157,9 @@ const executeStrategy = async () => {
 
   // 2. Try to detect devnet setup
   console.log('Basic test parameters are missing, Auto Setup Started');
-  const devnetStrategy = await localDevnetDetectionStrategy();
+  const devnetStrategy = await localDevnetDetector.execute();
   if (devnetStrategy.isDevnet) {
-    if (devnetStrategy.isRS) {
-      console.log('Detected Devnet-RS');
-    } else {
-      console.log('Detected Devnet-PY');
-    }
+    console.log(devnetStrategy.isRS ? 'Detected Devnet-RS' : 'Detected Devnet-PY');
```

Comment:
@penovicp I think we can remove the Devnet-PY stuff?

Reply:
If we don't need it, I'll just log the rs devnet detection indication within the execute directly